src/libs/ajax/leonardo/Disks.ts

```diff
@@ -91,6 +85,23 @@ export interface RawGetDiskItem extends RawListDiskItem {
   formattedBy: string | null;
 }
 
+export interface LeoDisksDataClientDeps {
```

Comment:
having this type here creates a circular dependency.... it's technically passable since it's only a type... but I think we should shift this type definition to the data client modules.... it would be fine to have a v1 and v2 version of this type that happens to have the same shape.

Reply:
Fair. I noticed this, and figured I would resolve it when actually moving things into the client packages.